### PR TITLE
Py2 py3

### PR DESCRIPTION
**Pull request by vatsa287**
Signed-off-by: vatsa287 <i.mnshreevatsa@gmail.com>

### CODE REVIEW

File: cli/kubectl_kadalu/utils.py [hunk 1]
```diff
@@ -3,23 +3,28 @@
...
     """ Class for handling exceptions """
-    def __init__(self, rc, err):
-        super().__init__("error %d %s" % (rc, err))
-        self.returncode = rc
+    def __init__(self, returncode, err):
+        # if sys.version_info >= (3,):
+        #     super().__init__("error %d %s" % (returncode, err))
+        # else:
+        #     super(CommandError, self).__init__(u"error %d %s" % (returncode, err))
+        super(CommandError, self).__init__(u"error %d %s" % (returncode, err))
```
Comment (aravindavk) [inline cli/kubectl_kadalu/utils.py:26]: is this working for Python3 as well?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
```
Comment (aravindavk) [inline .github/stale.yml:1]: extra file

Comment (vatsa287) [reply .github/stale.yml:1]: this got added into staging area even though I didnt add it to staging area.. Ill try again

File: cli/kubectl_kadalu/utils.py (hunk 1 above)
Comment (vatsa287) [reply cli/kubectl_kadalu/utils.py:26]: yes anna its working
```
vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ pylint3 utils.py
No config file found, using default configuration

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ pylint utils.py 
No config file found, using default configuration

------------------------------------
Your code has been rated at 10.00/10

vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ python utils.py
vatsa@vatsa-VirtualBox:~/kadalu/cli/kubectl_kadalu$ python3 utils.py
```

Comment (aravindavk) [reply cli/kubectl_kadalu/utils.py:26]: Just try to run using Python3 once. (May be by changing the kubectl command to something else in code, which will cause CommandError)

**Review by amarts — CHANGES_REQUESTED** [review]
I still see stale.yml..  if you are still learning git commands, Recommend doing a fresh apply of changes on git master.

File: cli/kubectl_kadalu/utils.py (hunk 1 above)
Comment (vatsa287) [reply cli/kubectl_kadalu/utils.py:26]: utils.py compiled on both ver of py . main.py has import errors without the changes

**Comment by vatsa287** [comment]
@amarts I tried to make the file in its most recent form by using its commit id. Ill try again

**Comment by aravindavk** [comment]
Copy the Changed files to temp

```
cp cli/kubectl_kadalu/install.py  /tmp/
cp cli/kubectl_kadalu/main.py     /tmp/
cp cli/kubectl_kadalu/storage_add.py /tmp/
cp cli/kubectl_kadalu/utils.py /tmp/
```

Now Fetch the latest source

```
git checkout master
git fetch upstream
git merge upstream/master
```

Now freshly checkout new Branch,

```
git checkout -b python2_support_cli master
```

Copy files back to source tree

```
cp /tmp/install.py cli/kubectl_kadalu/install.py
cp /tmp/main.py cli/kubectl_kadalu/main.py
cp /tmp/storage_add.py cli/kubectl_kadalu/storage_add.py
cp /tmp/utils.py cli/kubectl_kadalu/utils.py
```

Commit and Update the Commit message

```
git add cli/kubectl_kadalu/install.py \
    cli/kubectl_kadalu/main.py \
    cli/kubectl_kadalu/storage_add.py \
    cli/kubectl_kadalu/utils.py
git commit -s
```

Push and create a new Pull request, and Abondan the old ones.

```
git push origin python2_support_cli
```

**Comment by amarts** [comment]
Fixed in #244

**Pull request closed without merging**